docs/conf.py

```diff
@@ -103,9 +103,6 @@
 plot_html_show_formats = False
 plot_formats = ["svg"]
 
-# Redirects for pages that have moved
-redirects = {}
```

Comment:
not sure, but this might mean you should delete `"sphinx_reredirects"` from the list above, too.

Reply:
I simply moved the redirects down below to a new section with a header attached

Reply:
thanks, I see it now